src/Plugin/Block/ActivityFinder4Block.php

```diff
@@ -25,6 +26,12 @@ class ActivityFinder4Block extends BlockBase implements ContainerFactoryPluginIn
 
   use EntityBrowserFormTrait;
 
+  const THEME_BOOTSTRAP_VERSION = [
+    'openy_carnation' => 4,
```

Comment:
nice
But let's put this in config, please

Reply:
@manachynskyi

Reply:
Fixed.